cosmos/providers/databricks/workflow.py

```diff
@@ -112,8 +112,8 @@ def create_workflow_json(self) -> dict[str, object]:
             "job_clusters": self.job_clusters,
             "max_concurrent_runs": self.max_concurrent_runs,
         }
-        full_json = merge(full_json, self.extra_job_params)
-        return full_json
+        merged_json = merge(default_json, self.extra_job_params)
```

Comment:
Looking at the constructor `__init__ ()`, `self.extra_job_params` could be `None` if it is not passed. We should initialise it to empty dictionary `{}` if it is `None`. Or else merge would give a `TypeError` here.

Reply:
@pankajkoti the class `DatabricksWorkflowTaskGroup` was already handling that - but I'll add it to the Operator so we're on the safe side.